package.json

```diff
@@ -10,6 +10,9 @@
     "semantic-release": "semantic-release",
     "travis-deploy-once": "travis-deploy-once"
   },
+  "timings-api": {
+    "api_version": "0.0.0"
```

Comment:
0.0.0 ?

Reply:
the field will be populated during deployment by a shell script